ci-operator/step-registry/openshift/e2e/test/multi-iteration/openshift-e2e-test-multi-iteration-commands.sh

```
@@ -0,0 +1,159 @@
+    export TEST_PROVIDER=vsphere;;
+external) export TEST_PROVIDER='{"type":"external"}' ;;
+*) echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"; exit 1;;
+esac
+
+mkdir -p /tmp/output
+cd /tmp/output
+
+# Build test args
+TEST_ARGS=""
+if [[ -n "${TEST_FOCUS}" ]]; then
+    TEST_ARGS="--run '${TEST_FOCUS}'"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)